R/mod_export_data.R

```diff
@@ -87,12 +87,31 @@ mod_export_data_ui <- function(id) {
       fluidRow(column(
         width = 12,
         tags$div(
-          align = "left",
-          class = "multicol",
-          checkboxGroupInput(inputId = ns("selected_columns"),
-                             label = "Select Columns to Export")
+          checkboxInput(inputId = ns("select_all_columns"),
+                        label = "Include All Columns", 
+                        value = FALSE))
+      )), 
+      fluidRow(
+        column(
+          width = 6,
+          tags$div(
+            checkboxGroupInput(inputId = ns("selected_columns"),
+                               label = "Select Columns to Export"))
+        ),
+        column(
+          width = 3,
+          tags$div(
+            checkboxInput(inputId = ns("include_all_columns"),
```

Comment:
This is one way to do it, but since we already have all the checkboxes listed, a single select/deselect all button would be better for the UI.